ios/BlurView.m

```diff
@@ -37,12 +37,13 @@ - (void)setBlurType:(NSString *)blurType
     } else {
         self.blurEffect = [UIBlurEffect effectWithStyle:UIBlurEffectStyleDark];
```

Comment:
As you see, The super class [UIBlurEffect  -get/set effectWithStyle] is private method. If use [[BlurAmount alloc] init], we can't initial the instance of BlurAmount  with particular style. 
`NS_CLASS_AVAILABLE_IOS(8_0) @interface UIBlurEffect : UIVisualEffect
+ (UIBlurEffect *)effectWithStyle:(UIBlurEffectStyle)style;
@end`

I have tried [[BlurAmount alloc] init] , class_createInstance ,but didn't work on iOS8.